scripts/create_role_molecule.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import yaml
+from jinja2 import Environment, FileSystemLoader
+
+
+PROJECT_DIR = Path(__file__).resolve().parent.parent
+SCRIPT_DIR = PROJECT_DIR / 'scripts'
+CI_TEMPLATES_DIR = PROJECT_DIR / 'ci' / 'templates'
+CI_CONFIG_DIR = PROJECT_DIR / 'ci' / 'config'
+ROLES_DIR = PROJECT_DIR / 'ci_framework' / 'roles'
+ZUUL_JOB_DIR = PROJECT_DIR / 'zuul.d'
+SOURCE_ROLES_DOCS_DIR = PROJECT_DIR / 'docs' / 'source' / 'roles'
```

Comment:
I'd use standard `os.path.join()`

Reply:
I think is okay since we are not doing low level path manipulation and this lib is included from Python 3.4